### PR TITLE
Inventory and Item Service layers

### PR DESCRIPTION
**Pull request by RukmeshS**
Completed CRUD for Inventory and Item layers

### CODE REVIEW

File: src/main/java/com/ty/ims/inventory_prject_boot/service/InventoryService.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		}
+		throw new NoSuchIdFoundException();
+	}
+
+	public ResponseEntity<ResponseStructure<Inventory>> serviceFindInvById(int id) {
+		Optional<Inventory> inventory3 = dao.findInventorybyid(id);
+		ResponseStructure<Inventory> responseStructure = new ResponseStructure<Inventory>();
+		if (inventory3.isPresent()) {
+			responseStructure.setStatus(HttpStatus.FOUND.value());
+			responseStructure.setMessage("Inventory Product Not Found");
```
Comment (Prashantbag) [inline src/main/java/com/ty/ims/inventory_prject_boot/service/InventoryService.java]: change the message

File: src/main/java/com/ty/ims/inventory_prject_boot/service/ItemService.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		if (optional.isPresent()) {
+			responseStructure.setStatus(HttpStatus.FOUND.value());
+			responseStructure.setMessage("Inventory Item Found");
+			responseStructure.setData(optional.get());
+			return new ResponseEntity<ResponseStructure<Item>>(responseStructure, HttpStatus.FOUND);
+		}
+		throw new NoSuchIdFoundException();
+	}
+
+	public ResponseEntity<ResponseStructure<Item>> serviceDelete(int id) {
```
Comment (Prashantbag) [inline src/main/java/com/ty/ims/inventory_prject_boot/service/ItemService.java]: change delete method name

**Review by Prashantbag — COMMENTED** [review]
change the pointed lines

File: src/main/java/com/ty/ims/inventory_prject_boot/service/InventoryService.java (hunk 1 above)
Comment (RukmeshS) [reply src/main/java/com/ty/ims/inventory_prject_boot/service/InventoryService.java]: Done

File: src/main/java/com/ty/ims/inventory_prject_boot/service/ItemService.java (hunk 2 above)
Comment (RukmeshS) [reply src/main/java/com/ty/ims/inventory_prject_boot/service/ItemService.java]: Done in next commit

**Review by Abhishek-GP — APPROVED** [review]

**Review by Prashantbag — APPROVED** [review]

**Review by Prashantbag — APPROVED** [review]

**Pull request merged**